packages/synapse-interface/pages/bridge/index.tsx

```diff
@@ -683,6 +704,14 @@ const BridgePage = ({
   - Only executes if token has already been approved.
    */
   const executeBridge = async () => {
+    analytics.track(`[Bridge] User ${address} initiates bridge`, {
+      originChainId: fromChainId,
+      destinationChainId: toChainId,
+      inputAmount: fromInput.string,
+      receivedAmount: bridgeQuote.outputAmountString,
```

Comment:
receivedAmount -> expectedReceivedAmount, in case later we want to merge data with on-chain finalized values.